src/main/java/GeoSpatialConverter/odsProcessorUtilities.java

```diff
@@ -0,0 +1,50 @@
+package GeoSpatialConverter;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.codehaus.jettison.json.JSONException;
+import org.codehaus.jettison.json.JSONObject;
+
+public class odsProcessorUtilities
+{
+    private static final String ODS_PROCESSOR_NAME = "coordinateConversion";
+    private static final String ODS_PROCESSOR_VERSION = "2.0.0";
```

Comment:
I really think we should grab these values from a config.properties file instead of having to change the source code. The properties can be read write off the file and returned without needing to get into the project and build. If it is forgotten in the code it could be easily changed on the properties file and not have to redeploy.

Reply:
Updated with new changes to review